### DISCUSSION TITLE
Does this plugin allow to do `Reference in new issue` for a selected line of code?

### DISCUSSION THREAD

**Discussion by dzintars** [pwntester/octo.nvim#381] [Q&A]
Quite often I see the thing which should be handled (TODO), but I have no time to do that right away. I would like to open an issue and point to the specific line of code.
Looking in the main Readme, i don't see that this is supported.

**Comment by pwntester** [comment]
Hi! Does #363 work for you (creating a permalink that you can later paste in a new issue?)

**Comment by williambdean** [comment]
I have a custom workflow for this by combining gitlinker: 

I use `<leader>cri` to create an issue from the selected text using the permalink from gitlinker.

https://github.com/wd60622/dotfiles/blob/c2f239bed4a8ad646e06c2c2537f06c628530cf1/nvim/lua/plugins/version-control.lua?plain=1#L73-L80

The actual functionality from GitHub of "creating a reference issue" isn't supported. However, it would be an interesting feature. I don't know if there is a specific graphql mutation for it. But with the comment urls would be required and there could be a way of replicating something similar
